### ISSUE TITLE
where the GEM module?

### ISSUE THREAD

**Issue by caiyujue** [alanlukezic/d3s#25]
I find  this code:"dist_map = self.create_dist(init_patch_crop.shape[0], init_patch_crop.shape[1])",but is that the "The target localisation channel (L)" mentioned in the paper?
In segm_net.py the code :
  " segm_layers = torch.cat((torch.unsqueeze(pred_sm[:, :, :, 0], dim=1),
                                    torch.unsqueeze(pred_pos, dim=1),
                                    dist), dim=1)",  It looks like this "The target localisation channel (L)" is fixed as the code "self.dist_map = dist_map",has the "self.dist_map" changed?and where ?
thanks~

**Comment by caiyujue** [comment]
maybe I should change the " params.segm_dist_map_type = 'center'  " to params.segm_dist_map_type = 'bbox'?

**Comment by caiyujue** [comment]
I changed the " params.segm_dist_map_type = 'center' ,but the performance has deteriorated in VOT2018,when it's center mode,lost_num=47,while bbox mode ,the lost_num = 132,and I test the tracker with the pre-trained  model  also encountered the same problem.

**Comment by alanlukezic** [comment]
The segm_dist_map_type parameter should be set to 'center' since the network was trained this way. During training the L channel is simulated from the ground-truth target position, while the inference is implemented so that the L channel remains constant since target is cropped on the position estimated with GEM.

**Comment by caiyujue** [comment]
thank you for your reply

**Comment by laisimiao** [comment]
@caiyujue
Duplicate of #21
